File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/recyclerview/ComplexRecyclerActivity.java
package com.jdroid.android.sample.ui.recyclerview;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class ComplexRecyclerActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return ComplexRecyclerFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/recyclerview/ComplexRecyclerActivity.kt
package com.jdroid.android.sample.ui.recyclerview

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class ComplexRecyclerActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return ComplexRecyclerFragment::class.java
    }
}
